apps/web/pages/auth/forgot-password/index.tsx

```diff
@@ -148,8 +148,9 @@ ForgotPassword.PageWrapper = PageWrapper;
 export const getServerSideProps = async (context: GetServerSidePropsContext) => {
   const { req, res } = context;
 
-  const session = await getServerSession({ req, res });
+  const session = await getServerSession({ req });
 
+  // @TODO res will not be available in future pages (app dir)
```

Comment:
will this throw in the future pages?

Reply:
yes, it will throw when auth page is migrated. (it will throw in both legacy and future pages, because we pass legacyContext to both of them)

Reply:
what is the proper replacement? doing the following operation in the middleware?

Reply:
yes, all code that needs to modify headers/cookies should be moved to middleware, because in pages headers/cookies are readonly and res is not available at all.